crates/router/tests/connectors/adyen_uk_ui.rs

```diff
@@ -132,6 +132,29 @@ async fn should_make_adyen_klarna_mandate_payment(
     Ok(())
 }
 
+async fn should_make_adyen_alipay_hk_payment(c: WebDriver) -> Result<(), WebDriverError> {
```

Comment:
Please avoid using single character variable names. You can take it up in a separate PR, should be fine.